src/System.Data.Common/src/System/DBNull.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+////////////////////////////////////////////////////////////////////////////////
+// Void
```

Comment:
Shoudl this be "DBNull" instead of "Void", or should this header be removed entirely (leaving the copyright in place)?